campaigns.json

```diff
@@ -1012,11 +1012,29 @@
   },
   "Run3Summer22DR": {
       "fractionpass": 0.95,
-      "go": true
+      "go": true,
+      "secondaries": {
+        "/MinBias_TuneCP5_13p6TeV-pythia8/Run3Summer22GS-124X_mcRun3_2022_realistic_v10-v1/GEN-SIM": {
+          "SiteWhitelist": [
+            "T1_US_FNAL_Disk",
+            "T2_CH_CERN"
+          ]
+        }
+      },
+      "secondary AAA": false
   },
   "Run3Summer22DRPremix": {
       "fractionpass": 0.95,
-      "go": false
+      "go": false,
+      "secondaries": {
+        "/Neutrino_E-10_gun/Run3Summer21PrePremix-Summer22_124X_mcRun3_2022_realistic_v11-v2/PREMIX": {
+          "SiteWhitelist": [
```

Comment:
You should use `SecondaryLocation` location for Neutrinos. (I know it's very confusing, this is planned to be improved)